babel.config.js

```diff
@@ -34,6 +34,7 @@ module.exports = {
     '@babel/plugin-transform-flow-strip-types', // for a bug in FlatList caused by adding @babel/plugin-proposal-class-properties
     '@babel/plugin-proposal-class-properties', // for ethers
     '@babel/plugin-transform-private-methods', // for ethers
+    'macros',
```

Comment:
Why is it needed?
I looked at https://github.com/celo-tools/celo-web-wallet/blob/master/.babelrc and https://github.com/Uniswap/wallet/blob/main/apps/mobile/babel.config.js they both use `typed-redux-saga` but I can't yet find this there as well.

Reply:
It seems it may be unnecessary.
Not that both the projects listed above import like this:
`import { call, put } from 'typed-redux-saga'`
instead of 
`import { call, put } from 'typed-redux-saga/macro'` like we did in this PR.

Can we do the same?
Maybe this will help with the code coverage issue too?

Reply:
Answering myself, after re-reading the README for https://github.com/agiledigital/typed-redux-saga 😄 

Not entirely sure about the overhead of not using `macros` but it seems it would be quite [lite](https://github.com/agiledigital/typed-redux-saga/blob/master/src/index.js).
If this helps with code coverage, I think we should probably favor it.

Reply:
Without macros all of our redux-saga-test-plan tests were failing with various issues. I couldn't get it to work. See branch here: https://github.com/valora-inc/wallet/pull/3991

Reply:
oh thanks, I found a way to make it work: https://github.com/valora-inc/wallet/pull/4002

Shown on a single test, but I think all test should be able to work similarly.

Reply:
Awesome! Thanks @jeanregisser , I'll try to get it working on this PR.